packages/backend/src/plugins/catalog.ts

```diff
@@ -90,8 +82,8 @@ export default async function createPlugin(
       fn: async () => {
         await provider.run();
       },
-      frequency: Duration.fromObject({ minutes: 5 }),
-      timeout: Duration.fromObject({ minutes: 10 }),
+      frequency: Duration.fromObject({ minutes: 1 }),
```

Comment:
Can you change it back to at least 5 minutes, 1 minute seems like really frequent and I can imagine it will exhaust the ratelimits quite fast.

Reply:
good catch, done.